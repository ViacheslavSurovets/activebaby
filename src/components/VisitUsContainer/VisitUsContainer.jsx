import React from 'react';
import { CustomButton } from '@components';
import {
  VisitUsContentContainer,
  VisitUsContentText,
  VisitUsContentTitle,
  VisitUsImageContainer,
  VisitUsWrapper
} from './styles';

const VisitUsContainer = () => (
  <VisitUsWrapper>
    <VisitUsImageContainer to={ '#' } />
    <VisitUsContentContainer>
      <VisitUsContentTitle>Title</VisitUsContentTitle>
      <VisitUsContentText>Lorem ipsum dolor sit amet, consectetur adipisicing elit. At autem consectetur cumque
        cupiditate, dolorem eaque, eos error est eum illo, ipsam labore modi nobis odit perferendis quidem similique
        temporibus voluptatum.</VisitUsContentText>
      <CustomButton><span>Button</span></CustomButton>
    </VisitUsContentContainer>
  </VisitUsWrapper>
);

export default VisitUsContainer;
