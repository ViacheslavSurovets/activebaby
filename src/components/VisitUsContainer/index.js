export { default as VisitUsContainer } from './VisitUsContainer';
