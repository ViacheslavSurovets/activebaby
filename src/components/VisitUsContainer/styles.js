import styled from 'styled-components';
import { Link } from 'react-router-dom';
import { visitUs } from '@assets/visitUs';
import { commonStyleText, commonStyleTitleText, flexCommon } from '@commonStyles';
import { mobile } from '@media';


export const VisitUsWrapper = styled.section`
  display: flex;
  flex: 1;
  ${ mobile }{
    flex-direction: column;
  }
`;

export const VisitUsImageContainer = styled ( Link )`
  background-image: url(${ visitUs });
  background-size: cover;
  width: 50vw;
  height: 30vw;
  &:hover {
    filter: opacity(75%);
  }
  ${ mobile }{
    width: 100vw;
    height: 50vw;
  }
`;

export const VisitUsContentContainer = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  width: 80%;
  margin: 0 auto;
`;

export const VisitUsContentTitle = styled.span`
   ${ flexCommon };
   ${ commonStyleTitleText };
   width: 80%;
   margin-bottom: 1rem;
`;

export const VisitUsContentText = styled.span`
  ${ flexCommon };
  ${ commonStyleText };
  width: 80%;
  margin-top: 1rem;
  margin-bottom: 1rem;
`;
