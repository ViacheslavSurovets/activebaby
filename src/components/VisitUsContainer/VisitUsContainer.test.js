import React from 'react';
import { shallow } from 'enzyme';
import VisitUsContainer from './VisitUsContainer';


describe ( 'VisitUsContainer', () => {

  it ( 'VisitUsContainer Snapshot', () => {
    expect ( shallow ( <VisitUsContainer /> ) ).toMatchSnapshot ();
  } );
} );
