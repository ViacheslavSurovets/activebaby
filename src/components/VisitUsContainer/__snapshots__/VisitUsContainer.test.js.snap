// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisitUsContainer VisitUsContainer Snapshot 1`] = `ShallowWrapper {}`;
